src/server/reqsNonPi.txt

```diff
@@ -0,0 +1,17 @@
+Flask==1.1.*
```

Comment:
Is there a more readable name for this file? Maybe something like `requirements-core`?

Reply:
My thinking is that most users can/will safely ignore it.  It's referenced in the docs for Other Operating Systems.

Reply:
That's true, but I think the comment still makes sense. Not a deal-breaker by any means.

Reply:
Seems like calling it "requirements-core.txt" will make it more likely that users will think they need to do something with it.